float32 threshold
---
float32 average_five
float32 circles
---
uint32 random_number
uint32 average_minute

